### PR TITLE
Point all URLs to new domain [docs]

### PR DESCRIPTION
**Pull request by ghiculescu**
See also https://github.com/hotwired/stimulus/issues/356

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,23 +34,23 @@ export default class extends Controller {
...
-See the [Installation Guide](https://stimulusjs.org/handbook/installing) for detailed instructions.
+See the [Installation Guide](https://stimulus.hotwire.dev/handbook/installing) for detailed instructions.
 
 ## Getting Help & Contributing Back
 
-Looking for the docs? Once you've read through the Handbook, consult the [Stimulus Reference](https://stimulusjs.org/reference/controllers) for API details.
+Looking for the docs? Once you've read through the Handbook, consult the [Stimulus Reference](https://stimulus.hotwire.dev/reference/controllers) for API details.
 
-Have a question about Stimulus? Connect with other Stimulus developers on the [Stimulus Discourse](https://discourse.stimulusjs.org/) community forum.
+Have a question about Stimulus? Connect with other Stimulus developers on the [Stimulus Discourse](https://discourse.stimulus.hotwire.dev/) community forum.
```
Comment (javan) [inline README.md:53]:
```suggestion
Have a question about Stimulus? Connect with other Stimulus developers on the [Hotwire Discourse](https://discuss.hotwire.dev/) community forum.
```

**Review by javan — CHANGES_REQUESTED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (ghiculescu) [reply README.md:53]: thanks - i probably could have checked the links instead of just find and replacing.

**Comment by ghiculescu** [comment]
I don't wanna jinx it, but I think github actions (specifically `yarn test`) is struggling with forks of this repo https://github.com/hotwired/stimulus/actions

**Pull request merged**